home.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>YoutTube - Custome</title>
+</head>
+<body>
+    <ul class="header">
+        <li ><a href="#">YoutTube</a></li>
+        <li style="float: right;"><a href="#">LogOut</a></li>
+    </ul>
+    <div class="side-navigation">
+        <ul class="side-nav">
+            <li><a href="#" style="font-weight: bold">Subscription Channels</a></li>
+            <li><a href="#">Channel-1</a></li>
+            <li><a href="#">Channel-2</a></li>
+            <li><a href="#">Channel-3</a></li>
+        </ul>
+        <section>
+            <div class="grid-1">
+                <div>
```

Comment:
- [ ] I think you were not supposed to use `div` again with out reason

Reply:
- [ ] It would be better to give an explanation for the requested changes
. Why isn't he/she supposed to use `div` tag?

Reply:
- [ ] I think you were not supposed to use `div` again without reason. You already have a parent `div `tag. Your code will look if you write DRY(Don't repeat yourself) code